### ISSUE TITLE
Fail to compile in CRA project due to optional chaining

### ISSUE THREAD

**Issue by zhangyouxin** [nervosnetwork/lumos#197]
OS: not relevant
version: 0.18.0-rc2

I have installed the dependencies by  `yarn install` , after that, when I ran `yarn start`, I got this

![image](https://user-images.githubusercontent.com/3870972/143045754-f50a59a8-2790-4c56-9c1a-75987efacd0b.png)

**Comment by zhangyouxin** [comment]
we need `@babel/plugin-proposal-optional-chaining` plugin to make it compile

**Comment by e00dan** [comment]
I think this is fixed by: https://github.com/nervosnetwork/lumos/pull/227

Can we get a new release @homura that would fix CRA support?

**Comment by homura** [comment]
> Can we get a new release @homura that would fix CRA support?

We'll create a new release this week, but if you need to use it right now, then I recommend you use `@ckb-lumos/lumos`, which is a new [root package](https://github.com/nervosnetwork/lumos/pull/193), and it's CRA@4.0.3 compatible, we can find an example [here](https://github.com/nervosnetwork/lumos/blob/develop/examples/config-manager.ts)


```
yarn add @ckb-lumos/lumos@latest
```

```ts
import { config } from '@ckb-lumos/lumos'

/* initialize config via AGGRON testnet */
config.initializeConfig(config.predefined.AGGRON4);
console.log("testnet address is:", helpers.generateAddress(script));
```

**Comment by homura** [comment]
We have released [0.17.0-rc7(for CKB2019)](https://github.com/nervosnetwork/lumos/releases/tag/v0.17.0-rc7) and [0.18.0-rc4(for CKB2021)](https://github.com/nervosnetwork/lumos/releases/tag/v0.18.0-rc4) to work with CRA@4.0.3. 

Feel free to reopen this PR when you find that there are still compatibility issues with CRA@4.0.3
